apps/meteor/app/videobridge/client/tabBar.tsx

```diff
@@ -38,7 +36,7 @@ addAction('bbb_video', ({ room }) => {
 						id: 'bbb_video',
 						title: 'BBB_Video_Call',
 						icon: 'phone',
-						template: templateBBB,
+						template: lazy(() => import('../../../client/views/room/contextualBar/VideoConference/BBB')),
```

Comment:
keep this as a const outside the function, otherwise it will be always re-rendering

Reply:
@ggazzo Fixed!